core/src/validator.rs

```diff
@@ -1448,7 +1444,7 @@ fn load_blockstore(
             &blockstore,
             config.account_paths.clone(),
             config.account_shrink_paths.clone(),
-            config.snapshot_config.as_ref(),
+            Some(&config.snapshot_config),
```

Comment:
I decided to lave this one alone since `load_bank_forks()` can proceed without a snapshot by starting at slot 0. Although, if we implemented the suggestion in https://github.com/solana-labs/solana/issues/25577 to make `solana-genesis` create a snapshot for slot 0, that would no longer be true.